app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/viewmodel/PledgedProjectsOverviewViewModel.kt

```diff
@@ -219,6 +241,20 @@ class PledgedProjectsOverviewViewModel(
         )
     }
 
+    private fun isTier2Type(viewType: PPOCardViewType): Boolean {
+        return when (viewType) {
+            PPOCardViewType.ADDRESS_CONFIRMED,
+            PPOCardViewType.PLEDGE_COLLECTED_REWARD,
+            PPOCardViewType.PLEDGE_COLLECTED_NO_REWARD,
+            PPOCardViewType.SUVERY_SUBMITTED_DIGITAL,
+            PPOCardViewType.SUVERY_SUBMITTED_SHIPPABLE,
+            PPOCardViewType.AWAITING_REWARD,
+            PPOCardViewType.PLEDGE_REDEMPTION,
+            PPOCardViewType.REWARD_RECEIVED -> true
+            else -> false
+        }
+    }
```

Comment:
will move the above function, as well as `isTier1Type` to an ext class in subsequent PRs